### DISCUSSION TITLE
How do you unban someone?

### DISCUSSION THREAD

**Discussion by Oo1Toyo1oO** [LBPUnion/ProjectLighthouse#849] [Q&A] [answered]
The dismiss option for the cases don't seem to change anything, the user doesn't get unbanned, i don't know any further so im asking here, does anybody know how its done or is it not a feature?

**Comment by sudokoko** [comment]
The user is likely being unbanned. Your browser cache might just be playing tricks on you - it happens to me. Try hard reloading the user's profile? Also make sure your instance is up to date.

**Reply by Oo1Toyo1oO** [reply]
They're not though, they even posted a screenshot showing they are still banned.

**Reply by sudokoko** [reply]
Let me look in to this. I'll let you know

**Reply by Oo1Toyo1oO** [reply]
Thx

**Reply by sudokoko** [reply, marked as answer]
I've been told that you need to set the user's role back to default to unban them. I would also highly recommend syncing your fork and updating your instance as you're 85 commits behind upstream, however that's just a suggestion.

**Reply by Oo1Toyo1oO** [reply]
The user doesn't show up in the user list anymore though.

**Reply by sudokoko** [reply]
You would do this from the admin panel's user page, just search for their username using CTRL+F

**Reply by Oo1Toyo1oO** [reply]
Oh i didn't know that thanks, also its at the very bottom, nice i know what you mean now, alright thx for the fast support!
